bower.json

```diff
@@ -8,7 +8,6 @@
   "license": "http://polymer.github.io/LICENSE.txt",
   "private": true,
   "dependencies": {
-    "platinum-sw": "polymerelements/platinum-sw#1 - 2",
```

Comment:
Normally they should be declaring this as their dependency too, but some users might anyways rely on this transitive dependency. We should at least do a minor bump to the version once this gets merged (aka to 2.1.0)